grpc-examples/build.gradle

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import org.apache.tools.ant.taskdefs.condition.Os
+
+apply plugin: 'java'
+
+description = 'Ballerina - gRPC Examples'
+
+def ballerinaModulePath = "${project.rootDir}/grpc-ballerina/"
+def ballerinaDistPath = "${ballerinaModulePath}/build/target/extracted-distributions/jballerina-tools-zip/jballerina-tools-${ballerinaLangVersion}"
+def ballerinaDist = "${buildDir}/target/ballerina-distribution"
+def examples = ["routeguide/server", "routeguide/client"]
+
+dependencies {
+    implementation group: 'org.ballerinalang', name: 'ballerina-lang', version: "${ballerinaLangVersion}"
+    implementation group: 'org.ballerinalang', name: 'ballerina-tools-api', version: "${ballerinaLangVersion}"
+    implementation group: 'org.ballerinalang', name: 'ballerina-parser', version: "${ballerinaLangVersion}"
+    implementation group: 'org.testng', name: 'testng', version: "${testngVersion}"
+}
+
+task copyDistribution(type: Copy) {
+    from ballerinaDistPath
+    into ballerinaDist
+}
+
+task copyPackageBala {
+    doLast {
+        copy {
+            from "${ballerinaModulePath}/build/cache_parent"
+            into "${ballerinaDist}/repo"
+            copy {
+                into("bala/ballerina") {
+                    from "bala/ballerina"
+                }
+            }
+            copy {
+                into("cache/ballerina/") {
+                    from "cache/ballerina"
+                }
+            }
+        }
+    }
+}
+
+clean {
+    examples.forEach { example ->
+//        delete "${projectDir}/${example}/target"
```

Comment:
is it ok if we remove this?

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/320/commits/72eec3e5674f02d8379b31a6ed049d5a05446777